src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/assist/CompletionPopupPanel.java

```diff
@@ -103,15 +103,14 @@ public void onPreviewNativeEvent(NativePreviewEvent previewEvent)
             if (previewEvent.getTypeInt() == Event.ONMOUSEDOWN)
             {
                Element targetEl = previewEvent.getNativeEvent().getEventTarget().cast();
-               if (!help_.getElement().isOrHasChild(targetEl) &&
-                  (container_ == null) || 
-                  (container_.getElement() == null) || 
-                  !container_.getElement().isOrHasChild(targetEl))
-               {
+               boolean isActiveClickTarget =
+                     (help_ != null && help_.getElement().isOrHasChild(targetEl)) ||
+                     (container_ != null && container_.getElement().isOrHasChild(targetEl));
```

Comment:
just to confirm we don't need the `container_.getElement() != null` -- I guess that is implied if `container_.getElement().isOrHasChild(targetEl)` is true

Reply:
Yeah, exactly!